projects/cashmere/src/lib/tabs/tab-set/tab-set.component.ts

```diff
@@ -68,11 +68,11 @@ export class TabSetComponent implements AfterContentInit {
     /** Zero-based numerical value specifying which tab to select by default, setting to `none` means no tab
      * will be immediately selected. Defaults to 0 (the first tab). */
     @Input()
-    get defaultTab(): string {
+    get defaultTab(): string | number {
         return this._defaultTab;
     }
 
-    set defaultTab(tabValue: string) {
+    set defaultTab(tabValue: string | number) {
         if (Number(tabValue) !== NaN || tabValue === 'none') {
```

Comment:
This is actually not correct, since `NaN !== NaN`.  Should use `isNaN()` instead

Reply:
```ts
if (!isNaN(+tabValue) || tabValue === 'none') {
```

Reply:
Oh I just saw the second post - is there a difference between `isNaN(+tabValue)` and `isNaN(Number(tabValue))`